internal/sdkprovider/service/project/util.go

```diff
@@ -0,0 +1,31 @@
+package project
+
+import (
+	"github.com/aiven/aiven-go-client"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// DetermineAccountIDPointerForRequest returns the account ID for a request.
+func DetermineAccountIDPointerForRequest(client *aiven.Client, d *schema.ResourceData) (*string, error) {
+	var accountID *string
+
+	ownerEntityID, ok := d.GetOk("owner_entity_id")
+	if ok {
+		ownerEntityID, err := schemautil.NormalizeOrganizationID(client, ownerEntityID.(string))
+		if err != nil {
+			return nil, err
+		}
+
+		if len(ownerEntityID) == 0 {
+			return nil, nil
+		}
+
+		accountID = &ownerEntityID
+	} else {
+		// TODO: Remove this when account_id is removed.
+		accountID = schemautil.OptionalStringPointer(d, "account_id")
+	}
+
+	return accountID, nil
```

Comment:
string is a pointer since it is a pointer in the request where the value of this function is used, this function is used to get an account ID that is going to be used in multiple requests